nixos/modules/services/x11/xserver.nix

```diff
@@ -409,6 +409,16 @@ in
           if possible.
         '';
       };
+
+      enableCtrlAltBackspace = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to enable the DontZap option, which binds Ctrl+Alt+Backspace
```

Comment:
But @ocharles's point still stands: calling it Zap doesn't tell me anything about what it does.